mmcv/runner/hooks/logger/neptune.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Open-MMLab. All rights reserved.
+from ...dist_utils import master_only
+from ..hook import HOOKS
+from .base import LoggerHook
+
+
+@HOOKS.register_module()
+class NeptuneLoggerHook(LoggerHook):
+
+    def __init__(self,
+                 init_kwargs=None,
+                 interval=10,
+                 ignore_last=True,
+                 reset_flag=True,
+                 with_step=True,
+                 by_epoch=True):
+        """Class to log metrics to NeptuneAI.
```

Comment:
Those docstring should be moved to line 9.

Reply:
done